resources/qml/pages/MessagingPage.qml

```diff
@@ -185,14 +257,43 @@ Page {
                         }
                     }
                 }
-
             }
+            MouseArea {
+                anchors.fill: parent
+
+                onClicked: {
+                    if (startsWith(type, "image"))
+                        pageStack.push(Qt.resolvedUrl("ImagePage.qml"),{ 'imgUrl': item.file })
+                    else if (startsWith(type, "video"))
+                        pageStack.push(Qt.resolvedUrl("VideoPage.qml"),{ 'path': item.file })
+                    else if (startsWith(type, "audio"))
+                        pageStack.push(Qt.resolvedUrl("VideoPage.qml"),{ 'path': item.file });
+                }
 
+                onPressAndHold: {
+                    item.openMenu();
```

Comment:
On my device, no menu opens. Only the other items went 'inactive' or 'greyed out'.

Reply:
Yes, exactly. But is this intended? On an image or video, one should be able to choose at lease 'Save' or even 'Copy'. Only the 'Send again' should be disabled if the sending failed.

Reply:
Resolved